crates/eframe/src/native/glow_integration.rs

```diff
@@ -1140,9 +1143,15 @@ impl GlutinWindowContext {
             let width_px = NonZeroU32::new(width_px).unwrap_or(NonZeroU32::MIN);
             let height_px = NonZeroU32::new(height_px).unwrap_or(NonZeroU32::MIN);
             let surface_attributes = {
-                use rwh_05::HasRawWindowHandle as _; // glutin stuck on old version of raw-window-handle
                 glutin::surface::SurfaceAttributesBuilder::<glutin::surface::WindowSurface>::new()
-                    .build(window.raw_window_handle(), width_px, height_px)
+                    .build(
+                        window
+                            .window_handle()
+                            .expect("failed to get window handle")
```

Comment:
We don't want crashes. Return errors instead. Here and everywhere.
Add a case to `eframe::Error` if needed